src/renderers/shared/fiber/ReactFiberUpdateQueue.js

```diff
@@ -204,7 +205,9 @@ function findInsertionPosition(queue, update) : Update | null {
 // we shouldn't make a copy.
 //
 // If the update is cloned, it returns the cloned update.
-function insertUpdate(fiber : Fiber, update : Update, methodName : ?string) : Update | null {
+function insertUpdate(fiber : Fiber, update : Update, methodName : string) : Update | null {
```

Comment:
Since `methodName` is now always a string, we can remove the `typeof` check on line 215.